web/widgets.py

```
@@ -135,3 +139,216 @@ def __init__(self, attrs=None):
+                    // Initial render
+                    renderItems();
+                }});
+            </script>
+        </div>
+        """.format(
+            name,
+            "".join(feature_items),
+            json.dumps(decoded_value),  # {0}  # {1}  # {2}
+        )
+
+    def value_from_datadict(self, data: Dict, files: Dict, name: str) -> List:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)